scripts/test/system.sh

```diff
@@ -22,10 +22,13 @@ function deploy_env_once() {
 }
 
 function _subctl() {
+    local ret
     # Print GHA groups to make looking at CI output easier
     echo "::group::Running 'subctl $*'"
     "${DAPPER_SOURCE}"/cmd/bin/subctl "$@"
+    ret=$?
     echo "::endgroup::"
+    return $ret
```

Comment:
If you happen to respin this, otherwise you can leave as is.

You can make this change to ensure it exits (GHA will be fine without a closing `::endgroup::` anyway) and the code will be tidier :)

```suggestion
    # Print GHA groups to make looking at CI output easier
    echo "::group::Running 'subctl $*'"
    "${DAPPER_SOURCE}"/cmd/bin/subctl "$@" || exit "$?"
    echo "::endgroup::"
```